packages/go/analysis/azure/queries.go

```diff
@@ -319,33 +317,33 @@ func OutboundControlDescentFilter(ctx *ops.TraversalContext, segment *graph.Path
 	return shouldDescend
 }
 
-func OutboundControlPathFilter(ctx *ops.TraversalContext, segment *graph.PathSegment) bool {
-	return !segment.Edge.Kind.Is(azure.MemberOf)
+func OutboundControlPathFilter(_ *ops.TraversalContext, segment *graph.PathSegment) bool {
+	return !(segment.Edge.Kind.Is(azure.MemberOf) || segment.Edge.Kind.Is(azure.Contains))
```

Comment:
Changes here is for AZContains traversal and not part of cleanup

Reply:
Minor nit here but `Kind.Is(...)` is variadic and the `||` is not needed.